exec/plugin-commands-script-runners/src/exec.ts

```diff
@@ -208,6 +210,9 @@ export async function handler (
           result[prefix].status = 'passed'
           result[prefix].duration = getExecutionDuration(startTime)
         } catch (err: any) { // eslint-disable-line
+          if (await isErrorCommandNotFound(params[0], err)) {
+            err.hint = buildCommandNotFoundHint(params[0], (await readProjectManifestOnly(opts.dir)).scripts)
+          }
```

Comment:
🤔 if I add an `else` after this locally, then the command properly throws `err` after setting `err.hint`. But without the `else`, we only get the `exitCode = err.exitCode`. Is there a reason to run the `if (!opts.recursive ...` check if we know the failure is from the command not being found?